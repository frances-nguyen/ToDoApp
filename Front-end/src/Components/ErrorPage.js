import React, { Component } from 'react'

class ErrorPage extends Component {
  render() {
    return <div>An error has occurred. I don&apos;t know what to do!</div>
  }
}

export default ErrorPage
