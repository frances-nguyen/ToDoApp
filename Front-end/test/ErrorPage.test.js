import React from 'react'
import ErrorPage from '../src/Components/ErrorPage'
import renderer from 'react-test-renderer'

describe('<ErrorPage /> rendering', () => {
    it('renders correctly', () => {
        let tree = renderer.create(<ErrorPage />).toJSON()
        expect(tree).toMatchSnapshot();
    })
})
