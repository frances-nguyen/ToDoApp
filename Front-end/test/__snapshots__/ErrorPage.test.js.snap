// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage /> rendering renders correctly 1`] = `
<div>
  An error has occurred. I don't know what to do!
</div>
`;
